docs/configuration.md

```diff
@@ -327,6 +328,29 @@ The url to use when connecting to ElectricityMapsFree. Defaults to
 "https://api.co2signal.com/v1/" but can be overridden in the config if needed
 (such as to enable integration testing scenarios).
 
+## Cache
+
+Frequent access to data sources could cause problems such as performance trouble
+or exceed rate limit. To avoid them, you can configure data cache like this:
+
+```json
+{
+  "EmissionsDataCache": {
+    "Enabled": true,
```

Comment:
I'm not sure whether the configuration should be here or included in carbonAwareVers.